code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -465,3 +476,94 @@ GLOBAL_LIST_INIT(abductor_recipes, list ( \
 
 /obj/item/stack/sheet/mineral/coal/ten
 	amount = 10
+
+/*
+ * Morphium
+ */
+
+/obj/item/stack/sheet/mineral/morphium
+	name = "Morphium"
+	desc = "A powerful material that mimics what the gray* use."
```

Comment:
```suggestion
desc = "A powerful material that emulates the behaviors of advanced, rare alloys."
```
I'm pretty sure we don't call them "grays" on cit.

Reply:
![image](https://user-images.githubusercontent.com/30435998/87240811-e631ac00-c3ea-11ea-8342-2c01a04bfb57.png)

Reply:
I'll see if I can reconceptualize it when I get home